src/Fixer/FunctionNotation/NoSpacesAfterFunctionNameFixer.php

```diff
@@ -65,6 +63,17 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
                 if (!$tokens[$possibleDefinitionIndex]->isGivenKind(T_FUNCTION)) {
                     $this->fixFunctionCall($tokens, $index);
                 }
+            } elseif ($tokens[$lastTokenIndex]->equalsAny($braceTypes)) {
+                $block = Tokens::detectBlockType($tokens[$lastTokenIndex]);
+                if (
```

Comment:
The rule doesn't state anything about how multi lines, so I value it as incomplete.
That is my problem with PSR12 in general, it adds some guides which could be amended on PSR1/2 if FIG would allow it, than PSR12 could introduce new rules for new PHP syntax that was not around the time PSR1/2 were made. But lets not focus on this to much, happy to change to whatever style that gets this PR merged.